datafusion/physical-plan/src/windows/window_agg_exec.rs

```diff
@@ -33,19 +29,21 @@ use crate::{
     ExecutionPlan, ExecutionPlanProperties, PhysicalExpr, PlanProperties,
     RecordBatchStream, SendableRecordBatchStream, Statistics, WindowExpr,
 };
-
 use arrow::array::ArrayRef;
 use arrow::compute::{concat, concat_batches};
-use arrow::datatypes::{Schema, SchemaBuilder, SchemaRef};
+use arrow::datatypes::SchemaRef;
 use arrow::error::ArrowError;
 use arrow::record_batch::RecordBatch;
 use datafusion_common::stats::Precision;
 use datafusion_common::utils::{evaluate_partition_ranges, transpose};
 use datafusion_common::{internal_err, Result};
 use datafusion_execution::TaskContext;
 use datafusion_physical_expr::PhysicalSortRequirement;
-
 use futures::{ready, Stream, StreamExt};
+use std::any::Any;
+use std::pin::Pin;
+use std::sync::Arc;
+use std::task::{Context, Poll};
```

Comment:
Can you move them back to the top?

Reply:
I would appreciate a lot if there is `rustfmt.toml` so we just need `cargo fmt` and the import style is consistent

Reply:
I agree. We should automate this

Reply:
Yeah, I agree as well -- if we care about a consistent location / import order / style of `use` statements, let's get it in some automated tool that can be run as part of CI rather than try to enforce it manually during code review

I often see PRs that rearrange the imports and I suspect it is some IDE setting. Having it inconsistent makes merge conflicts more common but otherwise is not a problem in my mind